pages/desktop/user.py

```diff
@@ -119,8 +119,15 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
+        if not email_checkbox.is_selected():
```

Comment:
Instead of using an `if` here, change it to a wait so that we actually wait for the checkbox to be not selected. If that wait times out then the test will fail which makes sense, as the state we expected was not met.